sys/include/net/eui48.h

```diff
@@ -56,6 +56,24 @@ static inline void eui48_to_eui64(eui64_t *eui64, const eui48_t *addr)
     eui64->uint8[7] = addr->uint8[5];
 }
 
+/**
+ * @name EUI-48 bit flags contained in the first octet
+ *
+ * @see IEEE 802-2001 section 9.2
+ * @{
+ */
+
+/**
+ * @brief Locally administered address.
+ */
+#define EUI48_LOCAL_ADMIN_FLAG  0x02
```

Comment:
Sorry, last minute thing I noticed. Shouldn't this also be called `EUI48_LOCAL_FLAG` for consistency? (You may squash immediatelly)

Reply:
Quoting the IEEE 802 spec:
> The Individual/Group (I/G) address bit

and
> The Universally or Locally administered (U/L) address

I think I disagree with you here as the address itself is not local, but the address is locally administered (shortened to `LOCAL_ADMIN`) whereas with the other case the address is a group address. Although I don't mind shortening it to `EUI48_LOCAL_FLAG` if you insist.

Reply:
I know that this is about the uniqueness of the address, not the scope ;-). I just think the functions to set/clear a flag and the flag should have matching names. Either we call the function to set it then `_set_local_admin()` as well or we change the name here. I think `local` is completely sufficient, as it is well documented above that `local` means "locally administered" and for those unaware of the distinction it won't matter that much anyways.

Reply:
> I know that this is about the uniqueness of the address, not the scope ;-).

I know that you're aware of that, but the reader might not be. I agree that the names should match. Anyway, enough discussion, You've convinced me. I've renamed the define